modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -240,22 +242,14 @@ const getDriverSpec = (settings: DriverSettings, browserName: string): DriverLoa
 
 const driverSetup = async (driver: WebdriverIO.Browser, settings: DriverSettings, debuggingPort: number): Promise<void> => {
   // Browsers have a habit of reporting via the webdriver that they're ready before they are (particularly FireFox).
-
   // setTimeout is a temporary solution, VAN-66 has been logged to investigate properly
-  await driver.pause(1500);
+  await new Promise(resolve => setTimeout(resolve, 1500));
 
   // Log driver details
   logDriverDetails(driver, settings.headless, debuggingPort);
 
-  // Some tests require large windows, so make it as large as it can be.
-  // Headless modes can't use maximize, so just set the dimensions to 1280x1024
-  if (settings.headless) {
-    await driver.setWindowSize(1280, 1024);
-  } else {
-    await driver.maximizeWindow();
```

Comment:
This is still important, remote testing browsers regularly open at a very small size and cause tests to fail